tinygrad/nn.py

```diff
@@ -18,6 +18,7 @@ def __call__(self, x):
     if self.track_running_stats:
       self.running_mean = (1 - self.momentum) * self.running_mean + self.momentum * batch_mean
       self.running_var = (1 - self.momentum) * self.running_var + self.momentum * batch_var
+      if self.num_batches_tracked is None: self.num_batches_tracked = Tensor.zeros(1, requires_grad=False)
```

Comment:
How could this ever happen?

Reply:
Good question, I got baffled by this too... I spent a good while trying to find where `num_batches_tracked` could become None to no avail, so I added this line as a quick workaround... I still have to investigate this issue...